src/main/resources/default-recheck.ignore

```diff
@@ -1 +1,13 @@
-# Ignore for recheck. Please do not delete, even if they appear to be empty
+# Ignore file for recheck. Please do not delete, even if it is empty.
+
+# You can ignore specific attributes like so: 
+# matcher: retestid=div-b4f23, attribute: font.*
+
+# To ignore whole subtrees, use e.g. xpath:
+# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+
+# To ignore attributes globally, use 
+# attribute=class
+
+# More details and examples can be found here:
+# https://github.com/retest/docs/blob/master/docs/recheck/how-ignore-works-in-recheck.md
```

Comment:
I would suggest to always link against the deployed documentation: https://retest.github.io/docs/recheck/how-ignore-works-in-recheck/.

Reply:
Yes please, and it's also way shorter...